staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -176,5 +187,6 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 	}
 	attr.VersionedObject = newVersionedObject
 	o.GetObjectDefaulter().Default(attr.VersionedObject)
+	attr.GetReinvocationContext().RecordWebhookMutation(h.Name)
```

Comment:
check if the patch actually mutated the object (if `objJS` vs `patchedJS` or `attr.VersionedObject` vs `newVersionedObject` differ semantically)